### ISSUE TITLE
Support for looping groups

### ISSUE THREAD

**Issue by realies** [CasparCG/Client#197]
I know there's a looping functionality for individual clips, but I'd really want to see that possible on a group level.

**Comment by dimitry-ishenko** [comment]
@realies what about just looping individual clips in the group?
Looping groups can be confusing. What if you have 3 clips of different length? Should the group wait for the last clip to finish before looping? Or what if you have 2 clips and one of them has Use Auto set?

**Comment by jimbobhunter** [comment]
Yes indeed looping groups please. So a group of clips that will start playing again when the last item in the group has finished. To have a playlist of VT's looping in a pre-show environment is essential. It's currently the only thing that has me looking for other clients.

**Comment by jimbobhunter** [comment]
Are there any plans to implement this feature?

**Comment by rasoul13** [comment]
Guys please add this feature

**Comment by redimongo** [comment]
would be good. As I cant loop a playlist
